src/labels/CustomLabels.labels

```diff
@@ -89,6 +89,14 @@
         <value>Male</value>
     </labels>
     <labels>
+        <fullName>PreferredEmailMatchMustExist</fullName>
+        <categories>Contacts</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>PreferredEmailMatchMustExist</shortDescription>
+        <value>The value selected in Preferred Email does not match any email fields. Values in Preferred Email must match a custom email field label.</value>
```

Comment:
This error happens when an admin adds a value to the "Preferred Email" field, but there is not actually a corresponding custom email with a matching label. When Preferred Email is selected we match the labels to the custom fields added by an administrator. If there is a match, the matching custom email fields value is copied to the standard Email field on the contact.